R/chartutils.R

```diff
@@ -663,21 +663,24 @@ setDateTickDistance <- function(date.labels, num.maxticks)
         # Check for monthly intervals which accounts for different number of days per month
         seconds.in.day <- 86400
         nmonth <- round(tmp.dist/(seconds.in.day * 30))
-        if (!is.null(num.maxticks) && n > num.maxticks) 
+        if (!is.null(num.maxticks) && n > num.maxticks)
```

Comment:
Would be good to document the returned object (and also input arguments). Why is `NULL` returned in some cases?